listenbrainz/listen.py

```diff
@@ -196,7 +197,7 @@ def to_influx(self, measurement):
         # add the user generated keys present in additional info to fields
         for key, value in self.data['additional_info'].items():
             if key not in Listen.SUPPORTED_KEYS:
-                data['fields'][key] = value
+                data['fields'][key] = escape(value)
```

Comment:
Did this because `additional_info` variables may have random newlines in them, which would lead to parse errors.